packages/storybook/config/storybook/jetPieThemeDark.js

```diff
@@ -1,39 +1,40 @@
-import { create } from '@storybook/theming/create';
+import { create, color, typography } from '@storybook/theming';
 import logo from '../../public/logo--pie.svg';
 
 export default create({
     base: 'dark',
 
-    colorPrimary: '#fff',
-    colorSecondary: '#2a3846',
+    // Storybook-specific color palette
+    colorPrimary: '#f36d00',
```

Comment:
Any chance of a screenshot to show the difference in these changes to the Storybook UI?

Reply:
Yep will attach screen shots good idea